static/js/components/widgets/Label.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { filenameFromPath } from "../../lib/util"
+
+export interface Props {
+  name?: string
+  value?: string | File | null
+}
+
+/**
+ * A component for showing a read-only label
+ */
+const Label: React.FC<Props> = props => {
+  const { value } = props
+  return (
+    <div className="form-group">
+      {value && !(value instanceof File) ? (
+        <input
+          className="form-control"
+          placeholder={filenameFromPath(value)}
```

Comment:
Question: Is there any reason you chose `placeholder` over `value` here? I don't think it really matters since it's not editable 🤷. Just curious.

Reply:
Updated in https://github.com/mitodl/ocw-studio/pull/1406/commits/85fe0710ff61844c61732f5e487e5f047ed0dce1.